tests/onegov/fsi/test_views_reservation.py

```diff
@@ -1,8 +1,35 @@
+import pytest
 from onegov.fsi.collections.course_event import CourseEventCollection
-from onegov.fsi.models import CourseReservation, CourseAttendee, CourseEvent
+from onegov.fsi.models import CourseReservation, CourseAttendee, CourseEvent, \
+    Course
 from onegov.user import User
 
 
+@pytest.mark.skip('Webstest ES configuration issue')
```

Comment:
When you run this test, you get an issue with the ES and the app configuration. Can you reproduce it and see a hint in the StackTracke?